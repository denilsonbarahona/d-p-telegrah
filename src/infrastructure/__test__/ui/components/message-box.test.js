import React from "react";
import { render } from "@testing-library/react";
import MessageBox from "../../../ui/components/message-box";

describe("MessageBox", () => {
  it("rendering without crashing", () => {
    const { queryByRole } = render(<MessageBox />);
    const Message = queryByRole("textbox");
    expect(Message).toBeInTheDocument();
  });

  it("to match snapshot", () => {
    const { container } = render(<MessageBox />);
    expect(container).toMatchSnapshot();
  });
});
