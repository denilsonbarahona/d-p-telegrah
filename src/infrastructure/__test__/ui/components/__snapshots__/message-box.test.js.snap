// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBox to match snapshot 1`] = `
<div>
  <div
    class="sc-bczRLJ dcecjv"
  >
    <button
      class="sc-gsnTZi YVAZS"
      type="button"
    >
      <svg
        fill="currentColor"
        height="20px"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 512 512"
        width="20px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M216.08 192v143.85a40.08 40.08 0 0080.15 0l.13-188.55a67.94 67.94 0 10-135.87 0v189.82a95.51 95.51 0 10191 0V159.74"
          fill="none"
          stroke-linecap="round"
          stroke-miterlimit="10"
          stroke-width="32"
        />
      </svg>
    </button>
    <textarea
      class="sc-dkzDqf dwSxGX"
      placeholder="Write a message"
    />
    <div
      class="sc-hKMtZM ermruk"
    >
      <svg
        fill="currentColor"
        height="20px"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 512 512"
        width="20px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="184"
          cy="232"
          r="24"
        />
        <path
          d="M256.05 384c-45.42 0-83.62-29.53-95.71-69.83a8 8 0 017.82-10.17h175.69a8 8 0 017.82 10.17c-11.99 40.3-50.2 69.83-95.62 69.83z"
        />
        <circle
          cx="328"
          cy="232"
          r="24"
        />
        <circle
          cx="256"
          cy="256"
          fill="none"
          r="208"
          stroke-miterlimit="10"
          stroke-width="32"
        />
      </svg>
      <svg
        fill="currentColor"
        height="20px"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 512 512"
        width="20px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M192 448h128m64-240v32c0 70.4-57.6 128-128 128h0c-70.4 0-128-57.6-128-128v-32m128 160v80"
          fill="none"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="32"
        />
        <path
          d="M256 64a63.68 63.68 0 00-64 64v111c0 35.2 29 65 64 65s64-29 64-65V128c0-36-28-64-64-64z"
          fill="none"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="32"
        />
      </svg>
    </div>
  </div>
</div>
`;
